src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -563,31 +585,46 @@ export const ChartValuesEditor = ({
 
     useEffect(() => {
         if (!manifestView && chartValuesList.length > 0 && (isDeployChartView || deploymentHistoryList.length > 0)) {
-            const filteredChartValues = chartValuesList
-                .filter((_chartValue) => _chartValue.kind === ChartKind.DEPLOYED && _chartValue.name !== appName)
-                .map((_chartValue) => {
-                    return {
+            const deployedChartValues = [],
+                defaultChartValues = []
+            for (let index = 0; index < chartValuesList.length; index++) {
+                const _chartValue = chartValuesList[index]
+                if (_chartValue.kind === ChartKind.DEPLOYED && _chartValue.name !== appName) {
+                    deployedChartValues.push({
                         label: _chartValue.name,
                         value: _chartValue.id,
                         appStoreVersionId: _chartValue.appStoreVersionId,
                         info: `Deployed on: ${_chartValue.environmentName}`,
                         kind: _chartValue.kind,
-                    }
-                })
+                        version: _chartValue.chartVersion,
+                    })
+                } else if (_chartValue.kind === ChartKind.DEFAULT) {
+                    defaultChartValues.push({
+                        label: _chartValue.name,
+                        value: _chartValue.id,
+                        appStoreVersionId: 0,
+                        info: '',
+                        kind: _chartValue.kind,
+                        version: _chartValue.chartVersion,
+                    })
+                }
+            }
             const deploymentHistoryOptionsList = deploymentHistoryList.map((_deploymentHistory) => {
                 return {
                     label: moment(new Date(_deploymentHistory.deployedAt.seconds * 1000)).format(Moment12HourFormat),
                     value: _deploymentHistory.version,
                     info: '',
+                    version: _deploymentHistory.chartMetadata.chartVersion,
                 }
             })
 
             setValuesForDiffState({
                 ...valuesForDiffState,
-                filteredChartValues,
+                deployedChartValues,
+                defaultChartValues,
                 deploymentHistoryOptionsList,
                 selectedVersionForDiff:
-                    deploymentHistoryOptionsList.length > 0 ? deploymentHistoryOptionsList[0] : filteredChartValues[0],
+                    deploymentHistoryOptionsList.length > 0 ? deploymentHistoryOptionsList[0] : deployedChartValues[0],
```

Comment:
We can extend this condition further to use the first default values if no deployed chart is present.